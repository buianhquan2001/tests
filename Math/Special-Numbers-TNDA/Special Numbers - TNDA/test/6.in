99
2899 2684
3695 3128
2871 1275
4376 1934
4214 1465
3419 2013
3384 3770
3910 4912
1091 2765
2127 2721
1622 3747
2316 1615
2518 1817
2142 1943
1849 3964
4442 4106
2707 848
3860 3823
2915 2674
509 1344
1757 2022
1346 4254
2706 3975
589 3176
4743 4147
1512 2299
800 2289
2915 4742
722 3095
889 1032
624 635
551 4951
2283 3020
1899 2600
1673 3933
3279 4570
2031 2174
1280 2412
4604 1715
546 3196
4774 540
3115 1727
2267 2077
4491 2445
3726 2667
4127 4041
1414 3436
2561 3748
783 2576
3611 2715
3316 2463
2618 1531
1354 1011
3731 2976
2032 670
2990 3644
4979 3723
814 3175
2390 2597
2099 1304
1280 4027
3143 928
2818 983
3677 2111
3471 819
1385 3878
2721 3227
2990 2379
2596 3616
1762 3948
4692 4756
1887 2874
3681 1041
2178 1257
1604 3784
2116 4135
4848 1886
4649 2733
2243 3304
3312 1030
2370 2714
528 3400
2968 1080
869 4208
4201 2231
3758 1600
1642 1108
1526 3027
3628 2817
4949 4321
508 3431
3161 2747
2019 4565
3699 1111
4258 1630
4914 1365
1364 1841
2569 4643
3028 1233
