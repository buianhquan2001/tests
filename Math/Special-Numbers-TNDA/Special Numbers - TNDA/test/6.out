508848972
685038682
632382518
46224865
838895651
522745160
637648432
615877075
514028418
841215778
71675320
838169564
267902449
664780061
887851144
575860776
834351836
273535904
846446034
246172592
710998235
225783199
100974833
501806075
828945725
74483309
676922192
912146344
266644931
622923322
993797142
773065002
573999425
225721958
258878409
838273562
299695720
899646420
127351791
466822762
855814106
519944343
211629597
234352706
394592517
304767199
709235200
728348537
210286212
460033229
392753248
356599137
787507842
297098613
417990089
94022897
644232092
86929138
393169726
626674425
619070633
102286839
606254828
263501538
357666841
54054405
797016015
192662788
455740018
204707449
909555824
478732346
737849666
658652164
475668765
548697894
667569432
152969996
972865182
528153222
210611084
208969428
214630523
925243049
486896101
949931495
311531892
991178701
956867625
767750897
259974026
689075497
10895867
893671551
902927144
766895846
149105440
908313502
465586020
